code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -763,6 +763,20 @@
 		PROPERTY_FIRE_PENETRATING	= 1
 	)
 
+/datum/reagent/napalm/deathsquad //version of fuel for dsquad flamers.
```

Comment:
What is precisely the point of adding an entirely new chemical if the previous one is only used by whiteout flamers?
This is simply adding unnecessary code bloat, just change the values of the original.

Reply:
To allow the old version to be separately used and balanced as a fuel-type.  If a dev tells me I'll remove it though 😄